src/tests/App.test.js

```diff
@@ -0,0 +1,27 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import data from './data';
+import App from '../App';
+
+const reducer = (
+  state = {
+    home: { home: data },
+  },
+) => state;
+
+const store = configureStore({ reducer });
+describe('App test', () => {
+  it('will test App rendering', () => {
+    render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <App />
+        </Provider>
+      </MemoryRouter>,
+    );
+  });
+});
```

Comment:
-  [ ]  According to the project requirements :

   - The App should have unit tests for several components of the app and some pure functions in Redux code (currently you only tested the navbar component and App). The App should have integration tests using React Testing Library. Please, kindly add that 🙏

Reply:
Thank you, I really appreciate the time you spend reviewing my project.
I added two additional tests for SelectedCity and DetailsPages respectively.